src/main/java/step5/view/Announcer.java

```diff
@@ -0,0 +1,50 @@
+package step5.view;
+
+import step5.domain.collections.AttemptResult;
+import step5.domain.collections.RaceResult;
+import step5.dto.CarWentResult;
+import step5.view.output.OutputChannel;
+
+import java.util.List;
+
+import static step5.constants.MessageConstant.*;
+
+public class Announcer {
+
+    private final OutputChannel outputChannel;
+
+    public Announcer(OutputChannel outputChannel) {
+        this.outputChannel = outputChannel;
+    }
+
+    public void announceFinalResult(RaceResult raceResult) {
+        announceResultPerLap(raceResult);
+        announceWinnersName(raceResult);
+    }
+
+    private void announceResultPerLap(RaceResult raceResult) {
+        outputChannel.printLine(NEW_LINE+RESULT_MESSAGE);
+        List<AttemptResult> attemptResults = raceResult.getAttemptResults();
+        attemptResults.forEach(attemptResult -> printAttemptResult(attemptResult));
```

Comment:
이미 아시고 계시겠지만,
```suggestion
        attemptResults.forEach(Announcer::printAttemptResult);
```
이렇게 줄일 수도 있다는거 참고만 해주세요:)